docs/TESTPLAN.md

```diff
@@ -11,86 +11,60 @@
 * (Create profile: <https://developer.mozilla.org/Firefox/Multiple_profiles>, or via some other method)
 * Navigate to _about:config_ and set the following preferences. (If a preference does not exist, create it be right-clicking in the white area and selecting New -> String or Integer depending on the type of preference)
 * Set `extensions.legacy.enabled` to `true`. This permits the loading of the embedded Web Extension since new versions of Firefox are becoming restricted to pure Web Extensions only.
-* Set `extensions.button_icon_preference.variation` to `kitten` (or any other study variation/branch to test specifically)
+* Set `extensions.shield-search-nudges.variation` to `doshow` (or any other study variation/branch to test specifically)
```

Comment:
Not sure if we just want to explicitly list the variation options here. I had to do a bit of searching to find https://github.com/mikedeboer/shield-search-nudges/pull/1/files#diff-6c0726bed22ca8873400ea204e76fd1dR73